include/QtNodes/internal/Definitions.hpp

```diff
@@ -13,7 +13,7 @@
 
 namespace QtNodes
 {
-Q_NAMESPACE_EXPORT(NODE_EDITOR_PUBLIC)
+NODE_EDITOR_PUBLIC Q_NAMESPACE
```

Comment:
I mean, maybe it compiles, but can we load dlls and run examples?   I need to fix unit-tests asap

Reply:
Yes, I've run both caculater and simple_node and added node

Reply:
I have now tested the latest simplify-qt-find branch more thoroughly. On windows 10, qt version 5.11.0.
The following sections are included:
- models create and delete
- connections create and delete
- copy paste 
- drag
- resize

And I ran all the accompanying examples and did their basic operations.